src/main/java/software/amazon/awssdk/crt/s3/S3MetaRequestResponseHandlerNativeAdapter.java

```diff
@@ -32,4 +32,6 @@ void onResponseHeaders(final int statusCode, final ByteBuffer headersBlob) {
     void onProgress(final S3MetaRequestProgress progress) {
         responseHandler.onProgress(progress);
     }
+
+    void onTelemetry(final S3RequestMetrics requestMetrics) { responseHandler.onTelemetry(requestMetrics);}
```

Comment:
super trivial: this format looks awful.🤮

We don't have lint/formatter for Java, but try to follow the same pattern from the file.